### ISSUE TITLE
"Configure Keybindings" action appears in quick pick toolbar but does nothing when clicked in a11y dialog for notebooks

### ISSUE THREAD

**Issue by joyceerhl** [microsoft/vscode#223329]
Testing #222240

In this context when there are no unassigned keybindings, perhaps the action should be disabled or not added?

![Image](https://github.com/user-attachments/assets/77de3719-faee-41b8-843b-b32036316093)
